admin/acceptance_tests/user_test.py

```diff
@@ -150,3 +166,39 @@ def test_selenium(self, dbsession, selenium):
         dbsession.expire(user)
         assert user.username == 'new_name_éôù'
         assert user.email == 'new_email'
+
+    # in order to make this work, had to install selenium gecko driver
+    @pytest.mark.usefixtures("selenium", "selenium_app")
+    def test_delete_selenium(self, dbsession, selenium):
+        from c2cgeoportal_commons.models.main import User
+        user_id = dbsession.query(User). \
+            filter(User.username == 'babar_13'). \
+            one().id
+        selenium.get('http://127.0.0.1:6543' + '/user/')
+        elem = selenium.find_element_by_xpath("//div[@class='infos']")
+        assert 'Showing 1 to 10 of 23 entries' == elem.text
+        main_window_handle = selenium.current_window_handle
+        elem = selenium.find_element_by_xpath("//button[@title='Refresh']/following-sibling::*")
+        elem.click()
+        elem = selenium.find_element_by_xpath("//a[contains(.,'50')]")
+        elem.click()
+        elem = selenium.find_element_by_xpath("//a[contains(@href,'" +
+                                              str(user_id) +
+                                              "/edit')]/following-sibling::*")
```

Comment:
Are you sure that delete will stay the edit following-sibling, this is hard to understand, is it not possible to get the delete button directly ?

Reply:
Need to add a ref for this in template at c2cgeoform template side, no sure that we want to.

Reply:
We will have tu put a class on it for the icon, this could be addressed in this PR too.

Reply:
Icon don't carry id ! e.g. user.id.